src/Components/Blazor/WebAssembly.Authentication/src/Interop/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "microsoft.aspnetcore.components.webassembly.authentication",
```

Comment:
Is the intention to publish this as a package to NPM?

If not we should drop quite a few of the fields from here as they imply that we are doing. For example, reduce description to `"Not a public package"`, remove `repository`, `keywords`, `license`, etc.

Reply:
It looks like this is not intended to be used as a public package, since no `main` is specified.